domains.js

```diff
@@ -0,0 +1,72 @@
+const google = require('googleapis');
+const sheets = google.sheets('v4');
+
+function getDomains(username) {
+    const credentials = credentialConfiguration();
+    const request = {
+        spreadsheetId: credentials.TASK_SHEET_ID,
+        range: 'A:ZZZ', // extreme range to get whole spreadsheet
+        auth: credentials.API_KEY
```

Comment:
As noted earlier, this should probably be listed in `.env.example`. I think it might also be a good idea to to make this `GOOGLE_API_KEY` or something so people don’t confuse it with a key for `web-monitoring-db`.